packages/blade/src/components/Spinner/BaseSpinner/__tests__/BaseSpinner.native.test.tsx

```diff
@@ -34,6 +34,30 @@ describe('<BaseSpinner />', () => {
     expect(toJSON()).toMatchSnapshot();
   });
 
+  it('should render low contrast BaseSpinner with right label', () => {
+    const { toJSON } = renderWithTheme(
+      <BaseSpinner
+        accessibilityLabel="Loading"
+        contrast="low"
+        label="Loading"
+        labelPosition="right"
+      />,
+    );
+    expect(toJSON()).toMatchSnapshot();
```

Comment:
Suggestion: it might be better to assert for the presence of the labels explicitly instead of snapshot for these tests.

Reply:
I prefer snapshot here since it will also store the exact styles and for the label. If someone changes the prop of Text for the label from `subdued` to `normal`, we wouldn't be able to catch it with our tests unless we have a snapshot of it